code/game/area/Space Station 13 areas.dm

```diff
@@ -202,6 +204,7 @@ var/list/teleportlocs = list()
 	icon_state = "syndie-ship"
 	requires_power = 0
 	has_gravity = 1
+	noteleport = 1

```

Comment:
Ah, no. But!! computer-based-teleporters still function normally wherever they are built. You just can't teleport to beacons that are in anti-teleport zones.